src/utils/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios"
```

Comment:
Thanks for putting these into a separate file as suggested! 🙌 
You can also make things _even neater_ by using axios baseURL. Take a look at the notes at the bottom of this page if you wanna implement it 👍 
https://notes.northcoders.com/courses/js-front-end/axios